backend/wmg/pipeline/marker_genes.py

```diff
@@ -0,0 +1,89 @@
+import json
+import logging
+import os
+
+import pandas as pd
+import tiledb
+
+from backend.cellguide.pipeline.computational_marker_genes.computational_markers import MarkerGenesCalculator
+from backend.cellguide.pipeline.ontology_tree import get_ontology_tree_builder
+from backend.wmg.data.schemas.marker_gene_cube_schema import marker_genes_schema
+from backend.wmg.data.snapshot import (
+    CELL_COUNTS_CUBE_NAME,
+    EXPRESSION_SUMMARY_DEFAULT_CUBE_NAME,
+    MARKER_GENES_CUBE_NAME,
+    PRIMARY_FILTER_DIMENSIONS_FILENAME,
+    WmgSnapshot,
+)
+from backend.wmg.data.utils import log_func_runtime
+from backend.wmg.pipeline.constants import (
+    EXPRESSION_SUMMARY_AND_CELL_COUNTS_CUBE_CREATED_FLAG,
+    EXPRESSION_SUMMARY_DEFAULT_CUBE_CREATED_FLAG,
+    MARKER_GENES_CUBE_CREATED_FLAG,
+    PRIMARY_FILTER_DIMENSIONS_CREATED_FLAG,
+)
+from backend.wmg.pipeline.errors import PipelineStepMissing
+from backend.wmg.pipeline.utils import (
+    create_empty_cube_if_needed,
+    load_pipeline_state,
+    write_pipeline_state,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@log_func_runtime
+def create_marker_genes_cube(*, corpus_path: str):
```

Comment:
i don't think we need the `*` decorator here if there's only one param

Reply:
I will open a separate PR to remove that from all the different "create" functions :D